src/app/app.module.ts

```diff
@@ -23,6 +24,7 @@ import { NgbModule } from "@ng-bootstrap/ng-bootstrap";
     FavoriteModalComponent,
     FavoriteItemListComponent,
     SearchBarComponent,
+    SortComponent,
```

Comment:
You are still not integrating this, so you don't need to add the component (any of them tbh) to random modules.

Probably this won't be here in the future because we will split the code properly.